consensus/istanbul/backend/backend.go

```diff
@@ -299,7 +307,13 @@ func (sb *Backend) Address() common.Address {
 // Close the backend
 func (sb *Backend) Close() error {
 	sb.delegateSignScope.Close()
-	return sb.valEnodeTable.Close()
+	if err := sb.valEnodeTable.Close(); err != nil {
```

Comment:
We should still attempt to close the other table if the first fails. Returning the concatenation of the two error strings would be reasonable if both fail.